openmetadata-spec/src/main/resources/json/schema/analytics/basic.json

```diff
@@ -1,5 +1,5 @@
 {
-    "$id": "https://open-metadata.org/schema/analytics/basic.json",
+    "$id": "/analytics/basic.json",
     "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
@aniketkatkar97 why are you making the URL change?

Reply:
the `json2ts` script is not working properly, which is causing the build failure in the main. The cause was found to be that while resolving the refs, errors were thrown as `Error: Key definitions not in schema object at {path}`. We were suspecting the that it is not getting the json file properly through the URL. So trying an approach to give relative path instead. Wich seems to have solved the issue.
cc: @chirag-madlani @akash-jain-10

Reply:
@sureshms @harshach  can we merge this changes?

Reply:
@chirag-madlani and @aniketkatkar97, why did this problem start in the first place? We need a full URL so that JSON can be resolved by the external tools.

Reply:
@sureshms we have verified the full build with changes, can you point me to where exactly full URL needed?

Reply:
@chirag-madlani changing this is not an option "$id": "https://open-metadata.org/schema/analytics/basic.json"
that's the reason we said no to the previous PR
My input was to block the URL fetch from QuickType to remove the https://open-metadata.org/schema/ from the $id before passing it to the QuickType or fix the QuickType to ignore it and make a local resolution